components/project-onboarding/workspace.tsx

```diff
@@ -67,24 +67,6 @@ const Workspace = ({
                 <ArrowTopRightOnSquareIcon className="w-4 h-4 ml-1" />
               </button>
             </GoToSpace>
-
-            <form
-              action="/api/flatfile/sync-records"
-              method="post"
-              onSubmit={() => handleSubmit()}
-            >
-              <button
-                onClick={() => toast.loading("Syncing...")}
-                disabled={isSubmitting}
-                type="submit"
-                className={`${
-                  isSubmitting ? "hover:cursor-not-allowed" : ""
-                } border border-project-onboarding inline-flex items-center justify-center rounded-md px-4 py-2 text-sm font-medium text-project-onboarding hover:bg-project-onboarding hover:text-white`}
-              >
-                {buttonText}
-                <ArrowPathIcon className="w-4 h-4 ml-1" />
-              </button>
-            </form>
```

Comment:
Do we need to cut anything else like handleSubmit or unused vars?

Reply:
removed unneeded vars and logic. handleSubmit and other funcs were being used in the `setupSpace` component